fusor-ember-cli/app/controllers/openstack.js

```diff
@@ -1,8 +1,10 @@
 import Ember from 'ember';
 
 export default Ember.Controller.extend({
-  needs: ['undercloud-deploy'],
-  stepNumberOpenstack: Ember.computed.alias("controllers.undercloud-deploy.stepNumberOpenstack"),
+  needs: ['deployment', 'undercloud-deploy'],
+
+  stepNumberOpenstack: Ember.computed.alias("controllers.deployment.stepNumberOpenstack"),
   disableTabRegisterNodes: Ember.computed.alias("controllers.undercloud-deploy.disableTabRegisterNodes"),
-  disableTabAssignNodes: Ember.computed.alias("controllers.undercloud-deploy.disableTabAssignNodes")
+  disableTabAssignNodes: Ember.computed.alias("controllers.undercloud-deploy.disableTabAssignNodes"),
+  validOpenStack: true //TODO
```

Comment:
I think you'd just need the controller and compute roles assigned to a profile and their counts > 0.  If you're at this point in the workflow, you can probably (?) assume that the undercloud exists, since you would have been making API calls to the undercloud to get this far.